src/inference/rejection.js

```diff
@@ -22,6 +21,17 @@ module.exports = function(env) {
       maxScore: 0,
       incremental: false
     });
+
+    this.throwOnError = options.throwOnError !== undefined ? options.throwOnError : true;
+    // the value of options.probe is the min sample rate
+    this.probe = options.probe;
+    if (this.probe) {
+      // not throw error in probe mode
+      this.throwOnError = false;
+    }
+    this.numSamplesBak = options.samples;
```

Comment:
OK. Such renaming is good. I guess this part is confusing because it counts down rather than counts up?